client/configuration/config.js

```diff
@@ -1,5 +1,5 @@
 angular.module('config', [])
 
-.constant('ENV', {RequiresAuth:false,etcdHost:'127.0.0.1',etcdPort:'4001',hobknobHost:'localhost',hobknobPort:'3006',categories:[{id:0,name:'Simple Features',description:'Use when you want your feature to be either on or off'},{id:1,name:'Domain Features',description:'Use when you want your features to be toggled separately for different domains (e.g. com, couk, fr, ...)',values:['com','couk','de','fr']},{id:2,name:'Locale Features',description:'Use when you want your features to be toggled separately for different locales (e.g. en-GB, en-US, fr-FR, ...)',values:['en-GB','en-US','fr-FR','de-DE']}],plugin:{path:'../server/examplePlugin.js'}})
```

Comment:
the change in that line is just: RequiresAuth:true. 
Do you want me to remove it so that the client-side is reverted to Anonymous or is it because we would like to offer the default unauthenticated inside repo?